### PR TITLE
Config with pure python + setup GHA.

### PR DESCRIPTION
**Pull request by icemac**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,42 +1,62 @@
...
+    coverage-python-version
+    zope.testrunner
 commands =
-    coverage erase
-    coverage combine
+    coverage run -m zope.testrunner --test-path=src []
     coverage html
-    coverage xml
-    coverage report --fail-under=93
+    coverage report -m --fail-under=0
```
Comment (mgedmin) [inline tox.ini]: If we had a --fail-under=93, we should keep it.

File: tox.ini [hunk 2]
```diff
@@ -1,42 +1,62 @@
...
 
-[testenv:flake8]
-basepython = python3.6
-deps =
-    flake8
-    flake8-html
-    flake8-debugger
```
Comment (mgedmin) [inline tox.ini:40]: I wonder if flake8-debugger is something we might want to add to the meta template.

(Probably not; if we have 100% test coverage, any stray `pdb.set_trace()` calls will get caught anyway.)

**Review by mgedmin — APPROVED** [review]
LGTM, but maybe let's not drop the coverage threshold so drastically?

File: tox.ini (hunk 1 above)
Comment (icemac) [reply tox.ini]: Thank you for digging this up.

File: tox.ini (hunk 2 above)
Comment (icemac) [reply tox.ini:40]: As we have code reviews we will catch the other half. 😁

**Comment by icemac** [comment]
Thank you.

**Pull request merged**